### ISSUE TITLE
Help exits at top of screen, doesn't clear before displaying another

### ISSUE THREAD

**Issue by vr8hub** [PowerShell/PowerShell#6516]
Steps to reproduce
------------------

```powershell
help about_profile
```
`q` to exit.

Then immediately do it again.
```powershell
help about_profile
```

Expected behavior
-----------------
On 5.1, the first `q` exits the screen at the bottom. The text of the help is still on-screen, but the cursor and the input line are on the last line of the screen.
The second `help` (and anything else you might do) displays normally, scrolling or clearing the screen contents before displaying.

Actual behavior
---------------
On pwsh, the first `q` exits the screen at the top. The text of the help is still on-screen, but the cursor and the input line are the first line of the screen. This means that the output of anything else you do going forward gets mixed in with the help text that is left on-screen.

The second `help` doesn't clear the screen before it paints the text, resulting in garbled text. Here are a few lines from the top of the screen after the second `help`:

```powershell


ABOUT PROFILES
    Searching Help for about_profile ...
    [                                                                                                                        ]
SHORT DESCRIPTION
Describes how to create and use a PowerShell profile.
Describes how to create and use a PowerShell profile.

LONG DESCRIPTION
LONG DESCRIPTION
You can create a PowerShell profile to customize your environment and to
You can create a PowerShell profile to customize your environment and to.
add session-specific elements to every PowerShell session that you start.
A PowerShell profile is a script that runs when PowerShell starts. You can
A PowerShell profile is a script that runs when PowerShell starts. You cand
```

(To be clear, this is a screen handling issue, not a help or about_profile issue.)

Environment data
----------------

I did not load any profiles in either environment (5.1 or 6.x) for the purposes of this test. Both environments were started fresh, and I did not load any modules, etc. The "help about_profile" were the first commands I issued in both sessions.

Both consoles are 132x50, using Consolas 14pt.

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.2
PSEdition                      Core
GitCommitId                    v6.0.2
OS                             Microsoft Windows 10.0.16299
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by SteveL-MSFT** [comment]
Seems to happen due to setting `$outputencoding = [System.Console]::OutputEncoding`.  If you manually

> get-help about_profile | more

The issue doesn't repro.

**Comment by vr8hub** [comment]
It does (repro) for me. The below makes no difference whatsoever.

And I don’t know how it could (make a difference) — Get-Help isn’t doing the rendering, it’s just displaying what’s in the file. What is in the about*.help.txt file(s) should be already rendered text (and is, on every other about I’ve looked at, including at least one other one with a table, i.e. about_special_characters. The problem is with the about_profile.help.txt file; the text isn’t properly rendered in that file.

I created an issue in Powershell-Docs for this.

(On a different note, I tried cut-and-pasting (with Ctrl-V) your below line into the pwsh session, and it doesn’t work. And it turns out ctrl-V doesn’t work from anywhere. "Enable ctrl key shortcuts" is checked on the shortcut/session. Works fine in 5.1.)
